components/catalogs/back/src/Application/Service/TargetTypeConverter.php

```diff
@@ -15,7 +15,11 @@ final class TargetTypeConverter
     private const PIM_ATTRIBUTE_TYPES = [
         'string' => [
             'pim_catalog_text',
+            'pim_catalog_number'
         ],
+        'number' => [
+            'pim_catalog_number'
+        ]
```

Comment:
```suggestion
        'number' => [
            'pim_catalog_number',
        ],
```

We need to enforce this on the CI